classes/ObjectModel.php

```diff
@@ -261,7 +261,7 @@ public function __construct($id = null, $id_lang = null, $id_shop = null, $trans
 
         if ($id) {
             $entity_mapper = ServiceLocator::get('\\PrestaShop\\PrestaShop\\Adapter\\EntityMapper');
-            $entity_mapper->load($id, $id_lang, $this, $this->def, $this->id_shop, self::$cache_objects);
+            $entity_mapper->load($id, $this->id_lang, $this, $this->def, $this->id_shop, self::$cache_objects);
```

Comment:
Bald move! 😅

Reply:
Yeah, but in the end, it makes more sense to me

Reply:
yep, totally